packages/generators/generators/lib/plops/utils/get-formatted-date.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = (date = new Date()) => {
+  return new Date(date.getTime() - date.getTimezoneOffset() * 60000).toJSON().replace(/[-:T]/g, "").replace(/\....Z/, "");
+};
```

Comment:
```suggestion
module.exports = (date = new Date()) => {
  return new Date(date.getTime() - date.getTimezoneOffset() * 60000)
    .toJSON()
    .replace(/[-:]/g, '.')
    .replace(/\....Z/, '');
};
```

Reply:
I believe this change plus the one in migration.js matches the documentation [here](https://docs.strapi.io/developer-docs/latest/developer-resources/database-migrations.html), but please double-check me on that :)

Reply:
`2023.02.09T14.34.33.name-of-my-migration.js`

The purpose of this pull request was to avoid confusing the order of migrations. I have no idea about the order of this date format in windows and linux.